documentation-website/Writerside/topics/Breaking-Changes.md

```diff
@@ -24,6 +24,20 @@
   -- Starting from version 0.57.0
   INSERT INTO TEST DEFAULT VALUES
   ```
+  
+* The fields `defaultValueFun`, `dbDefaultValue`, and `isDatabaseGenerated` have been consolidated into a single field, 
+  `default`, of type `ColumnDefault`. The previous setup with the three separate fields often led to inconsistencies 
+  and confusion regarding their various combinations.
```

Comment:
In my opinion, this document should only include information that is relevant to the user. It's good to justify the rationale behind our decisions, but 2 of these fields are actually internal and will mean nothing to most users.

For the list below, a user would never have used `dbDefaultValue != null` or `isDatabaseGenerated == true` in their code, so I don't think they need to be noted.
The breaking changes that should be mentioned are how:
```
Column.defaultValueInDb() -> Column.databaseDefaultExpression()
Column.isDatabaseGenerated() -> Column.hasDatabaseDefault()

// this part I'm not sure how to suggest a migration for...
Column.defaultValueFun -> Column.default ???
```

Also, is there a migration element that is maybe missing?
What if for example the field `defaultValueFun` was being invoked somewhere in a user's logic like:
```
Column.defaultValueFun?.invoke() -> ???
// what would this become?
// Column.defaultValue() or Column.clientDefaultValue() or Column.clientDefaultValueOrExpression() ?
// would the user need to know the underlying ColumnDefault and cast?
```
Does there maybe need to something like `ColumnDefault.invoke()` that returns the result of invoking the Kotlin function if it exists or throws an error otherwise?